addons/website/static/src/js/menu/content.js

```diff
@@ -646,7 +646,7 @@ var EditMenuDialog = weWidgets.Dialog.extend({
                 levels[menu.depth] = (levels[menu.depth] || 0) + 1;
                 var menuFields = this.flat[menu.id].fields;
                 menuFields['sequence'] = levels[menu.depth];
-                menuFields['parent_id'] = menu['parent_id'] || this.rootMenuID;
+                menuFields['parent_id'] = parseInt(menu['parent_id']) || menu['parent_id'] || this.rootMenuID;
```

Comment:
Or actually why not doing that in python in the `save` method? (So if someone else if calling `save()` from custom module or whatever and also passing strings, they will benefit from the fix too)

Reply:
> Why not just [... always int]

Because the requests for creating new nodes contains actual strings such as `"new-5"` which is interpreted on the python side to obtain new ids.